### PR TITLE
Feat: Add Responsive Landing Page

### PR DESCRIPTION
**Pull request by aniketxparihar**

### CODE REVIEW

File: src/Pages/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+.landing__page__heading{
+    margin:1rem 0rem;
+}
+.landing__page__heading--text{
+    font-size: 6rem;
+    color:#000000;
+}
+.landing__page__description--text{
+    font-size: 3rem;
+    margin: 0rem 0rem 0rem 0rem;
```
Comment (shraddha15-creator) [inline src/Pages/LandingPage/LandingPage.css:31]: Why are you not giving just ` margin: 0; ` here? It would also work

**Review by shraddha15-creator — APPROVED** [review]
UI looks clean
ClassNames are also good.

File: src/Pages/LandingPage/LandingPage.css (hunk 1 above)
Comment (aniketxparihar) [reply src/Pages/LandingPage/LandingPage.css:31]: Forgot to remove thanks!

**Review by narharikale — APPROVED** [review]
clear and understandable code

**Pull request merged**